frontend/pages/software/ManageSoftwarePage/ManageSoftwarePage.tsx

```diff
@@ -566,8 +566,7 @@ const ManageSoftwarePage = ({
   const isCollectingInventory =
     !searchQuery &&
     !filterVuln &&
-    !isAnyTeamSelected &&
-    !page &&
```

Comment:
With recent changes, this now always returns false as page is now always set to a number (default 0) so had to update the logic accordingly